ocaml/xapi/xapi_host.ml

```diff
@@ -1403,7 +1403,8 @@ let install_server_certificate ~__context ~host ~certificate ~private_key
   (* sever all connections done with stunnel *)
   Xapi_mgmt_iface.reconfigure_stunnel ~__context
 
-let emergency_reset_server_certificate ~__context =
+let reset_server_certificate ~__context ~host =
```

Comment:
`~host` is unused, but needs to be defined because of how the definition machinery